client/js/lounge.js

```diff
@@ -40,13 +40,17 @@ $(function() {
 		storage.set(name, state);
 	}
 
-	$("#windows").on("click", function(e) {
-		const isOpen = slideoutMenu.isOpen();
+	// If sidebar overlay is visible and it is clicked, close the sidebar
+	$("#sidebar-overlay").on("click", () => {
+		slideoutMenu.toggle(false);
+		storeSidebarVisibility("thelounge.state.sidebar", false);
+	});
 
-		if ((isOpen && $(window).outerWidth() < utils.mobileViewportPixels) || $(e.target).is(".lt")) {
-			slideoutMenu.toggle(!isOpen);
-			storeSidebarVisibility("thelounge.state.sidebar", !isOpen);
-		}
+	$("#windows").on("click", "button.lt", () => {
+		const isOpen = !slideoutMenu.isOpen();
```

Comment:
@xPaw, if we can click on this button, doesn't it mean that it's closed anyway?

Reply:
You can see both on desktop.

Reply:
Duh 😴